infra_tooling_account/app.py

```diff
@@ -17,6 +66,12 @@
 
 TAGS = {"project_name": PROJECT_NAME, "stage_name": STAGE_NAME}
 
+(
+    general_settings_reader,
+    monitoring_groups_settings_reader,
+    recipients_settings_reader,
+) = read_settings()
+
```

Comment:
let's keep it like this for the current PR.
In future, I'd say:

settings_reader = LocalFSSettingsReader(config_folder)
settings = settings_reader.read_and_prepare()
settings.validate()

and, then, settings (which is an object containing all 3 settings structs) are passed to each stack construct as a param

@NastassiaPaskrobka what do you think?

Reply:
I had an idea of creating settings_provider module and putting it along with settings_validator under src/lib/settings/cdk folder - to make sure this would be used by CDK only.

And there i declared the same read function (naming can be discussed) which accepts config folder path, reads and validates all the files - and provides settings readers as the result. Tested this approach - works fine, and no such read_settings functions code duplication.

Reply:
`(
    general_settings_reader,
    monitoring_groups_settings_reader,
    recipients_settings_reader,
) = settings_provider.read_settings("../config/settings")`

That's all it takes to call it in app.py